packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -97,7 +97,14 @@ type UIEventConfigMap = {
 
 // Extend UIEventConfig with additional event-specific configuration
 export type UIEventConfig = UIEventConfigMap[keyof Omit<EventHandlerMap, 'change'>]
-  | (UIEventConfigMap['change'] & { includeInitialDisabledState?: boolean });
+  | (
+    UIEventConfigMap['change'] & {
+      // (Default: true) Whether to include default state of radio/input/select elements when surfaces are mounted.
+      includeInitialDefaultState?: boolean,
```

Comment:
why do we need to separate this from normal change event? We do fire this event with change event as well.

Reply:
never mind, I saw the WWW diff.